grails-app/controllers/org/broadinstitute/orsp/api/NewConsentGroupController.groovy

```diff
@@ -128,6 +132,12 @@ class NewConsentGroupController extends AuthenticatedController {
         render([consent: consent] as JSON)
     }
 
+    def findUseRestriction() {
+        def restriction = DataUseRestriction.findByConsentGroupKey(params.consentKey)
+        Collection<String> duSummary = consentService.getSummary(restriction)
```

Comment:
This can fail, i.e., return nothing if the consent doesn't have one created for it yet.